woocommerce-services.php

```diff
@@ -514,6 +500,17 @@ public function load_textdomain() {
 			load_plugin_textdomain( 'woocommerce-services', false, dirname( plugin_basename( __FILE__ ) ) . '/i18n/languages' );
 		}
 
+		public function jetpack_on_plugins_loaded() {
+			$jetpack_config = new Automattic\Jetpack\Config();
+			$jetpack_config->ensure(
+				'connection',
+				array(
+					'slug' => WC_Connect_Jetpack::JETPACK_PLUGIN_SLUG,
+					'name' => __( 'WooCommerce Shipping & Tax', 'woocommerce-payments' ),
```

Comment:
`'woocommerce-payments'` should probably be `'woocommerce-services'`

Reply:
```suggestion
					'name' => __( 'WooCommerce Shipping & Tax', 'woocommerce-services' ),
```
ope